### PR TITLE
vencord but dir

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110408&drop=dogfoodAlpha

### CODE REVIEW

**Comment by solomoncyj** [comment]
[Policy] Blocking-Issue

**Comment by stephengillie** [comment]
Was the `Blocking-Issue` label for the `Specify the install root` prompt? I can manually specify this - what should be used?

image

File: manifests/v/Vendicated/Vencord/location/1.2.10/Vendicated.Vencord.location.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+PackageIdentifier: Vendicated.Vencord.location
+PackageVersion: 1.2.10
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Discord.Discord
+Installers:
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/v/Vendicated/Vencord/location/1.2.10/Vendicated.Vencord.location.installer.yaml:8]:
```suggestion
- Architecture: x86
```
This package wouldn't run until I changed this in the VM's local manifest.

File: manifests/v/Vendicated/Vencord/location/1.2.10/Vendicated.Vencord.location.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  InstallerType: exe
```
Comment (stephengillie) [inline manifests/v/Vendicated/Vencord/location/1.2.10/Vendicated.Vencord.location.installer.yaml:9]:
```suggestion
  InstallerType: portable
```

Comment (stephengillie) [reply manifests/v/Vendicated/Vencord/location/1.2.10/Vendicated.Vencord.location.installer.yaml:9]: This appears to be a portable package:
```
Using UserConfig
Checking if C:\Users\User\AppData\Roaming\Vencord\dist exists: Yes
Is Dev Install:  false
Fetching https://api.github.com/repos/Vendicated/Vencord/releases/latest
Checking if C:\Users\User\AppData\Local\Discord\app-1.0.9013\resources exists: Yes
Checking if C:\Users\User\AppData\Local\Discord\app-1.0.9013\resources\app exists: No
Checking if C:\Users\User\AppData\Local\Discord\app-1.0.9013\resources\app.asar is directory: No
Found Discord install at  C:\Users\User\AppData\Local\Discord
Vencord Installer cli v1.2.10 (89c8345)
Usage of C:\Users\User\AppData\Local\Temp\WinGet\Vendicated.Vencord.location.1.2.10\VencordInstallerCli.exe:
  -branch string
        Select the branch of Discord you want to modify [default|stable|ptb|canary]
  -install
        Install Vencord on a Discord install
  -install-openasar
        Install OpenAsar on a Discord install
  -location string
        Select the location of your Discord install
  -uninstall
        Uninstall Vencord from a Discord install
  -uninstall-openasar
        Uninstall OpenAsar from a Discord install
  -update
        Update your local Vencord files
```

**Comment by solomoncyj** [comment]
> Was the `Blocking-Issue` label for the `Specify the install root` prompt? I can manually specify this - what should be used?
> 
> image
> 

Your discord install

**Pull request closed without merging**